update

```diff
@@ -416,3 +424,375 @@ DIRS.forEach(function (dir) {
     })
   }
 })
+
+class IPLDMock {
```

Comment:
That's a lot of code for something that was a single line before. It nearly doubled the size of the script :) And we'd need to check if its license is compatible with ours (GPLv3).

Honestly, I'd rather stay with the old `ipld` than maintain something like this. There must be a way to do the same thing with the new libs that replaced it.